config/application.rb

```diff
@@ -29,6 +29,7 @@ class Application < Rails::Application
     config.autoload_paths << "#{root}/lib"
```

Comment:
My reading of https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#config-autoload-lib-and-config-autoload-lib-once is that we can remove this (and maybe the next line) because we're now using `autoload_lib` instead.

Reply:
Addressed in 36d8a4d. The next line does appear to be needed.